interface/globals.php

```diff
@@ -131,7 +131,10 @@ function GetCallingScriptName()
                 $srcdir = "../library";
                 require_once("$srcdir/auth.inc");
             }
-            die("Site ID is missing from session data!");
+            // Site ID missing from session data
+            require_once("session_expired.php");
+            sessionExpired($web_root);
```

Comment:
Lets call this sessionBroken and script session_broken.php
(this is because site expiration should not lead to this; if it does then is a bug. things that should lead to this are bad things like php not having write privileges to php session data or when running out of drive space(since can't write to session then))